src/en/config-gce.md

```diff
@@ -1,255 +1,201 @@
-should select `APIs & auth`, and then select `Credentials` from the panel which
-will appear.
+The API is already enabled if the button displays 'Disable API'. If the button
+shows 'Enable API', clicking it may prompt you to set up a billing method (if
+not already done).
 
-![image showing main screen](media/config-gce04.png)
 
-On the credentials page itself, select 'Create new Client ID'
+## GCE project credentials file
 
-![image showing API page](media/config-gce05.png)
+A collection of credentials-related material will be required. This is obtained
+by downloading a file from the UI.
 
-A pop-up will appear asking which type of credential to generate. You *must* 
-select the `Service account` option - Juju needs these credentials to be able to
-create new instances.
+Return to the 'API Manager' and choose the 'Credentials' screen. By default you
+will be within the 'Credentials' tab. Click the 'New credentials' button and
+choose 'Service account key' among the 3 options available:
 
-![image showing account popup](media/config-gce06.png)
+![Create credentials dialog #1](./media/config-gce-api_manager_create_credentials-1.png)
 
+In the ensuing dialog, select 'Compute Engine default service account' and 'JSON'
+key type:
 
-## 5. Retrieve the credentials
+![Create credentials dialog #2](./media/config-gce-api_manager_create_credentials-2.png)
 
-Once you have clicked on the `Create Client ID` button, the GCE website will
-automatically generate a JSON file containing the credentials and download it
-through your browser. This file contains all the information required for Juju
-to connect to the GCE project.
-This can be used in two ways as described below.
-If for any reason you lose this information, you *cannot* regenerate the
-private key. You can however generate a new key by returning to the credentials
-page and clicking on the `Generate new JSON key` button, after which you may
-want to delete the original fingerprint entry from the list on the same page.
+Once the 'Create' button is pressed you will be prompted to download a file.
+This is the file we're after. Store it safely as this file cannot be
+regenerated (although a new one can easily be created).
 
+Place this file where the Juju client can find it. This may or may not be on
+the computer you downloaded the file to. We recommend the `~/.juju` directory.
+For the current example, the file is called `Juju-GCE-f33a6cdbd8e3.json` (based
+on our project name of 'Juju-GCE'. Let it be put here:
 
-## 6. Enter the credentials into your environments.yaml file
+`/home/ubuntu/.juju/Juju-GCE-f33a6cdbd8e3.json`
 
-There are two ways to add this information to the environments.yaml file used by
-Juju, both of which are explained here:
+!!! Warning: Due to
+[LP #1533790](https://bugs.launchpad.net/juju-core/+bug/1533790) make a copy of
+the original file and put it in another location. Edit the original file by
+removing the lines containing these keywords: 'project-id', 'auth-uri',
```

Comment:
are they lines? I thought it was just a blob. Suggest  s/lines/sections for clarity